backend/tests/appointmentDetails.test.ts

```diff
@@ -68,6 +68,18 @@ describe('appointment details', () => {
     // @ts-expect-error ts-migrate(2339) FIXME: Property 'getAppointment' does not exist on type '... Remove this comment to see the full error message
     whereaboutsApi.getAppointment = jest.fn().mockResolvedValue(testAppointment)
```

Comment:
Should it be calling whereaboutsApi to get an appointment here? Suppose this is not a VLB, so maybe yes